pom.xml

```diff
@@ -190,12 +191,15 @@
     <version.com.ibm>11.5.0.0</version.com.ibm>
     <version.com.microsoft.sqlserver>mssql2005_sqljdbc_2.0</version.com.microsoft.sqlserver>
     <version.com.oracle>18.3.0.0</version.com.oracle>
+    <version.com.sun.messaging.mq.fscontext>4.6-b01</version.com.sun.messaging.mq.fscontext>
     <version.com.sybase.jConnect>6.0</version.com.sybase.jConnect>
     <version.commons-httpclient>3.1-jbossorg-1</version.commons-httpclient>
     <version.compiler.plugin>3.8.0-jboss-2</version.compiler.plugin>
     <version.doxia>1.8.1</version.doxia>
     <version.glassfish.json>2.0.1</version.glassfish.json>
     <version.hamcrest>2.2</version.hamcrest>
+    <version.hamcrest>1.3</version.hamcrest>
```

Comment:
Thanks @moulalis for your PR! Please use only one property for version.hamcrest (lines 200 and 201)

Reply:
The dependency hamcrest-library (version 1.3) was moved to hamcrest dependency (https://mvnrepository.com/artifact/org.hamcrest/hamcrest-library). I think we should use version 2.2, Thanks

Reply:
See PR I created https://github.com/jbosstm/narayana/pull/2086

Reply:
Thanks @marcosgopen, my bad I missed removing it.

Reply:
Thanks